.github/workflows/markdown-toc.yml

```diff
@@ -0,0 +1,23 @@
+name: Markdown TOC
+
+on:
+  push:
+    paths:
+      - README.md
+  pull_request:
+    paths:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          curl https://raw.githubusercontent.com/ekalinin/github-markdown-toc/master/gh-md-toc -o gh-md-toc
```

Comment:
Rather than downloading `master` we should get the file at some specific commit.

Reply:
Even better would be to use an action from the marketplace I guess.

Reply:
I don't think I could find one on the marketplace equivalent to this one, perhaps the author just didn't want to publish it (yet)...  this is the template for the action I found on the repo itself: https://github.com/ekalinin/github-markdown-toc#toc-generation-with-github-actions.

Reply:
Don't worry. `https://raw.githubusercontent.com/ekalinin/github-markdown-toc/17ec017/gh-md-toc` should do.